src/main/resources/templates/bamboo/c/sequentialRuns/3_build_and_run_structural_tests.sh

```diff
@@ -1,20 +0,0 @@
-#!/usr/bin/env bash
-
-# Actual build process:
```

Comment:
if we do not support sequential tests any more, we should make sure that instructors cannot choose this option when they choose C as programming language